reactdotnet/reactdotnet.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.8" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.9" />
```

Comment:
Shouldn't we go 2.1.*?

Reply:
> 
> 
> Shouldn't we go 2.1.*? 

we can, 2.0.9 was just the lowest recommended version. Should we use the latest (2.2.2)?